core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/restore/DatabaseRestoreHandler.java

```diff
@@ -77,7 +77,8 @@ protected Selectable doAccept(HandlerEvent<DatabaseRestoreRequest> event) {
             Set<String> gatewayFQDN = Collections.singleton(gatewayInstance.getDiscoveryFQDN());
             ExitCriteriaModel exitModel = ClusterDeletionBasedExitCriteriaModel.clusterDeletionBasedModel(stackId, cluster.getId());
             String rangerAdminGroup = rangerVirtualGroupService.getRangerVirtualGroup(stack);
-            SaltConfig saltConfig = saltConfigGenerator.createSaltConfig(request.getBackupLocation(), request.getBackupId(), rangerAdminGroup, true, stack);
+            SaltConfig saltConfig = saltConfigGenerator.createSaltConfig(request.getBackupLocation(), request.getBackupId(), rangerAdminGroup,
```

Comment:
I'm assuming you're just restoring all DBs by default since we're currently not exposing any functionality for skipping restoring certain DBs.
Have you verified though that running this restore, where it will attempt to restore all DBs, won't result in any errors due to the backup for some DBs not existing?

Reply:
This was not a problem. the restore_db.sh handles the missing backup by just skipping it.